src/frontend/js/components/SearchFilterValueParent/SearchFilterValueParent.tsx

```diff
@@ -12,13 +12,13 @@ import { SearchFilterValueLeaf } from '../SearchFilterValueLeaf/SearchFilterValu
 
 const messages = defineMessages({
   ariaHideChildren: {
-    defaultMessage: 'Hide child filters',
+    defaultMessage: 'Hide additional filters for {filterValueName}',
     description:
       'Accessibility message for the button to hide children of the current filter',
     id: 'components.SearchFilterValueParent.ariaHideChildren',
   },
   ariaShowChildren: {
-    defaultMessage: 'Show child filters',
+    defaultMessage: 'Show more filters for {filterValueName}',
```

Comment:
or subfilters ? I'm thinking `sous-filtres` in french?

Reply:
"Show more filters for Mathematics" // "Voir plus de filtres pour Mathématiques"
"Show Mathematics subfilters" // "Voir les sous-filtres de Mathématiques"

The first one sounds simpler to me (even though it's less precise to the ears of Richie-experts like ourselves)

Reply:
According to the poll, your proposal is the best!